evidence/alin/0003-alin-retain-I-2024.md

```diff
@@ -0,0 +1,75 @@
+# Evidence-0002: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/12/19)                                                             |
+| **Submitted by**| Alin Dima                                                                                   |
+
+
+## Member details
+
+- Matrix username: `@alin:parity.io`
+- Polkadot address: `148f8D1P4CP2tV8JuaVHzEXQQgj3jBxEg3k9qZydPzkJjbQG`
+- Current rank: I
+- Date of initial induction: 2024-04-26
+- Date of last report: 2024/10/01
+- Area(s) of Expertise/Interest: Parachains consensus, Elastic Scaling, Availability
+
+
+## Reporting period
+
+- Start date: 2024/10/01
+- End date: 2024/12/19
+
+
+## Evidence
+
+My contributions in these past months have been mostly centered around Elastic Scaling.
+
+### Elastic Scaling
+
+I have continued the implementation of [RFC103](https://github.com/polkadot-fellows/RFCs/pull/103), which enables having an untrusted collator
```

Comment:
Can you please explain a bit as to why this is important for Polkadots broader mission?  
It seems relevant to me, but i dont really understand what the impact is and why it would be important to work on.

Reply:
It's important for decentralisation and resilience. More motivation is included in the RFC text:
> For Elastic Scaling, it prevents anyone who has acquired a valid collation to DoS the parachain by providing the same collation to all backing groups assigned to the parachain. This can happen before the next valid parachain block is authored and will prevent the chain of candidates from being formed, reducing the throughput of the parachain to a single core.

If you agree, I wouldn't modify the text of the evaluation now, because there's already a referendum up for vote. Thanks for the feedback, I'll try to be more explicit in my next evaluation